### DISCUSSION TITLE
Exporting mesh generated by MOOSE

### DISCUSSION THREAD

**Discussion by tannhorn** [idaholab/moose#23730] [Q&A Meshing] [answered]
Hello, is it possible to export the mesh generated in MOOSE into e.g. the gmsh format? I have been looking through the documentation and the forum and haven't found anything, except for "_MOOSE supports reading and writing a large number of formats and could be extended to read more._" in https://mooseframework.inl.gov/application_usage/mesh_block_type.html

Thanks!

**Comment by GiudGiud** [comment, marked as answer]
Hello

No we do not have this writing capability, though we would be interested in the contribution.

Best,
Guillaume

**Reply by cpgr** [reply]
If the mesh isn't too complex I've had success using [meshio](https://pypi.org/project/meshio/), but it only does the mesh and not any associated variables etc.

**Reply by tannhorn** [reply]
Thanks, I will check it out
